astropy/io/fits/hdu/hdulist.py

```diff
@@ -1039,7 +1047,8 @@ def filename(self):
 
     @classmethod
     def _readfrom(cls, fileobj=None, data=None, mode=None, memmap=None,
-                  cache=True, lazy_load_hdus=True, **kwargs):
+                  cache=True, lazy_load_hdus=True, ignore_missing_simple=False,
```

Comment:
Per my previous comment I'm not sure we even need an option for this; absent a more generic "verification" framework I don't think there's ever any reason not to check this and not to consider it an error if the `SIMPLE` keyword is missing.